src/config.ts

```diff
@@ -44,7 +52,7 @@ export type BotConfigMap = {
 	filler?: FillerConfig;
 	fillerLite?: FillerConfig;
 	spotFiller?: FillerConfig;
-	trigger?: BaseBotConfig;
+	trigger?: FloatingMakerConfig;
 	liquidator?: LiquidatorConfig;
 	floatingMaker?: BaseBotConfig;
```

Comment:
shouldnt this be for `floatingMaker` not `trigger`?

Reply:
Ugh, you are correct.